src/services/auth/auth.test.tsx

```diff
@@ -39,6 +41,16 @@ describe("auth", () => {
       ]);
     });
 
+    it("should return an object with warning and status keys if the response contains these properties", async () => {
```

Comment:
Possible to include a test to express what the difference in these 2 properties are?

Reply:
i see. didn't know that this is a use for tests

Reply:
What I meant was to provide more details on what these properties mean and how they interact with each other. You've added a test showcasing that we only need `status` and not `warning`, but looking at the types, what if there's only `warning` and what if the response is empty?

These can either be defined in the tests or as comments in the defined type. Both provide a means to let future devs know what kind of results are expected. The tests explicitly showcase some possible values, e.g. is status a status code? is it a human readable string? The types on the other hand showcase which must exist in the response (i.e required vs optional).

Reply:
i see. looking at the backend code, status should always be returned. will update the tests and types accordingly